torchreid/integration/sc/openvino_task.py

```diff
@@ -91,72 +87,32 @@ def __init__(
             Good value is the number of available cores. Defaults to 1.
         :param device: Device to run inference on, such as CPU, GPU or MYRIAD. Defaults to "CPU".
         """
-        super().__init__(model_file, weight_file, device, num_requests)
-
-        if len(label_schema.get_labels(False)) == 1:
-            self.labels = label_schema.get_labels(include_empty=True)
-        else:
-            self.labels = label_schema.get_labels(include_empty=False)
-        self.empty_label = get_empty_label(label_schema)
-        self.multilabel = len(label_schema.get_groups(False)) > 1 and \
-                len(label_schema.get_groups(False)) == len(label_schema.get_labels(include_empty=False))
 
-        self.hierarchical = False
```

Comment:
I think that at the current stage of development the original OpenVINO inferencer should be restored in OpenVINO task, while omz-based inferencer could be utilized in deploy() method. 
The motivation is the following:
- these changes are breaking current functionality in SC: if someone create a project with hierarchical labels, this will cause a training pipeline failure, because after training the model is converted to IR and evaluated via OV task. So after merging this PR we would get shrinking of SC classification component capabilities. SC managers wouldn't be happy when this feature will disappear, since they love it for some reasons.
- we are going to improve hierarchical label handling by introducing multihead classifications models and more sophisticated postprocessing. OpenVINO task should be self-containing to enable experiments in that direction. At the same time basic scenarios (single/multilabel) will be backward compatible with the omz model wrapper.

Reply:
I didn't delete possibility to use hierarchical labels, I separated postprocessing for classification with any operations using labels. And I save old postprocessing in my current realization. Please take a look [PR](https://github.com/openvinotoolkit/training_extensions/pull/775)